src/lib/containers/evaluation_queues/EvaluationCard.tsx

```diff
@@ -118,15 +118,13 @@ export const EvaluationCard: React.FunctionComponent<EvaluationCardProps> = ({
                 utc={utc}
               />
               {permissions?.canSubmit && (
-                <div>
-                  <Button
-                    className="submit-button"
-                    type="primary"
-                    onClick={onSubmit}
-                  >
-                    Submit
-                  </Button>
-                </div>
+                <Button
```

Comment:
was wrapping around `<div>` to get `display:block` behavior, now the button itself has `display:block`